tools/mode2_decode.cpp

```diff
@@ -0,0 +1,106 @@
+// Quick and dirty tool to decode GlobalCache (GC) codes
```

Comment:
I think the description here doesn't match what the program does, nor who the author is. ;-)
Please update and maybe include a usage example.

My guess from skimming the code is it takes STDIN in the form of lines with "space nnnn" or "pulse nnnn" where `nnnn` is the duration. One per line etc. Correct?